packages/ui/packages/Carousel/src/core/utils.js

```diff
@@ -1,3 +1,10 @@
+/* eslint-disable no-useless-escape */
+/* eslint-disable array-callback-return */
+/* eslint-disable radix */
+/* eslint-disable no-plusplus */
+/* eslint-disable no-return-assign */
+/* eslint-disable no-param-reassign */
+/* eslint-disable consistent-return */
```

Comment:
combine this disables in one line, and do not disable every thing :|
please fix them, or at least do disable it for one line

Reply:
done